go.mod

```diff
@@ -3,9 +3,9 @@ module github.com/shipwright-io/cli
 go 1.15
 
 require (
-	github.com/onsi/gomega v1.10.1
+	github.com/onsi/gomega v1.10.3
 	github.com/pkg/errors v0.9.1
-	github.com/shipwright-io/build v0.3.0
+	github.com/shipwright-io/build v0.3.1-0.20210330182238-23d2672f2f61
```

Comment:
For `CLI`, do we want to follow nightly in our `main` branch and only depend on releases when we release `CLI`?
@adambkaplan @otaviof @qu1queee